web/elm/src/Login/Login.elm

```diff
@@ -55,7 +55,7 @@ viewLoginState userState isUserMenuExpanded isPaused =
                 ([ href "/sky/login"
                  , attribute "aria-label" "Log In"
                  , id "login-container"
-                 , onClick LogIn
+                 , onClick <| Click LoginButton
```

Comment:
Starting to notice all these `onClick <| Click`s. Obviously we can't fix that now, but it would be a nice thing to clean up if and when we move to our own Dom model.

Reply:
Yeah I got just about every place, but I think there are some places with mouseDown or places where Ctrl-click or Shift-click makes a difference, so I'd like to pull out a better abstraction to cover all those cases.